v21.2/bulk-delete-data.md

```diff
@@ -22,7 +22,7 @@ Exercise caution when batch deleting rows from tables with foreign key constrain
 Before reading this page, do the following:
 
 - [Install CockroachDB](install-cockroachdb.html).
-- [Start a local cluster](secure-a-cluster.html), or [create a {{ site.data.products.dedicated }} cluster](../cockroachcloud/create-your-cluster.html).
+- [Start a local cluster](secure-a-cluster.html) or [create a {{ site.data.products.serverless }} cluster](../cockroachcloud/create-a-serverless-cluster.html).
```

Comment:
Instead of using these, I'd suggest we use the 2 quickstart variations.

Start a local cluster: https://www.cockroachlabs.com/docs/cockroachcloud/quickstart.html?filters=local
Start a cloud cluster: https://www.cockroachlabs.com/docs/cockroachcloud/quickstart.html

I'd probably switch the order and start with cloud. Also, I think we can remove the "Install CockroachDB" link. If folks choose the local cluster option, they'll get there.